### ISSUE TITLE
Repo conflict with RPMfusion

### ISSUE THREAD

**Issue by firebadnofire** [natesales/repo#3]
RPMFusion ships the "q" package for Q (The code language) which conflicts with the "q" package from this repo

Suggestion/Solution:

Add a "priority=N" operator in /etc/yum.repos.d/natesales.repo

This solution has worked so that this repo's "q" package can download to my Alma Linux system

What such a config would look like:
![Screenshot_20240912-090824.png](https://github.com/user-attachments/assets/1c2f0259-586c-4412-b047-32818758f9ee)

**Comment by firebadnofire** [comment]
It may also be beneficial to add this to the Apt repo too. This is what that config would look like in `/etc/apt/preferences.d/natesales`:
![Screenshot 2024-09-12 094346](https://github.com/user-attachments/assets/86f3b86a-eb62-406b-8d0d-6afa32b948cc)

**Comment by huyz** [comment]
yeah `q` is an obviously terrible name. Have we not learned from the `golang` insanity?
